static/css/section/_download.scss

```diff
@@ -1,225 +1,20 @@
-    }
-
-    @media only screen and (min-width : $breakpoint-medium) {
-      .strip-inner-wrapper {
-        padding: 0;
-      }
-    }
-
-    #main-content .row-hero {
-        margin-bottom: 0;
-        border-bottom: 0;
-    }
-    .gap-from-top {
-        margin-top: 20px;
-    }
-    .chinese-download {
-        margin-bottom: 20px;
-    }
-    .row-list-download {
-        padding-top: 0;
-
-        .box-highlight {
-            margin-bottom: 20px;
-        }
-    }
-    p.ubuntu-upgrade {
-        background: $light-grey url('#{$asset-server}f68488b1-picto-upgrade-warmgrey.svg') 20px center no-repeat;
-        background-size: 40px 40px;
-        border-top: 1px solid #e3e3e3;
-        border-radius: 0 0 4px 4px;
-        display: block;
-        margin-bottom: 0;
-        padding: 20px 0 20px 84px;
-        margin: 0 -20px -41px -20px;
-    }
-}
-
-.download-server-thank-you-linuxone #main-content .row-hero {
-  border-bottom: 1px dotted $warm-grey;
-}
-
-
-.download-cloud-home #main-content .row form ul li{
-  color: inherit; // overrides the ubuntu orange default on these pages
-}
-
-html.no-svg,
-html.opera-mini {
-    .download p.ubuntu-upgrade {
-        background-image: url('#{$asset-server}fa83e8f4-pictorgram-upgrade-44x44.png');
-    }
-}
-
-.download-home {
-    .row-hero p {
-        font-size: 1.21875em;
-    }
-}
-
-.download-desktop-home {
-  .row .download-description {
-    border: 0;
-    margin-right: 0;
-  }
-}
-
-.download-alternative-downloads {
-    #main-content .row-hero {
-        border-bottom: 1px dotted $warm-grey;
-    }
-}
-
-.download-desktop-upgrade {
-    #main-content .row-hero {
-        padding-bottom: 0;
-    }
-    .alternative-options {
-        margin-top: 20px;
-    }
-}
-
-.chinese-download {
-    padding-top: 0;
-}
-
-
-/* heros */
-.download-alternative-downloads .row-hero,
-.download-desktop-upgrade .row-hero,
-.download-ubuntu-kylin-zh-CN .row-hero {
-  background: none;
-}
-
-.download-ubuntu-kylin-zh-CN .lang-switch,
-.download-ubuntu-kylin .lang-switch {
-    position: absolute;
-    top: 20px;
-    right: 40px;
-    color: #fff;
-    z-index: 999;
-}
-
-.download-ubuntu-kylin-zh-CN .row-zh-CN .box-highlight {
-    box-sizing: border-box;
-}
-
-.download-ubuntu-kylin-zh-CN .row-zh-CN .box-highlight h2,
-.download-ubuntu-kylin-zh-CN .row-zh-CN .box-highlight p {
-    padding-right: 20px;
-}
-
-/*	@section download > desktop/server & cloud
--------------------------------------------------------------- */
-.download .box-highlight.no-margin-bottom {
-  margin-bottom: 0px;
-}
-
-.download-desktop .windows-8 {
-  padding-top: $gutter-width;
-}
+@charset 'UTF-8';
 
-.download .desktop.box-highlight {
-  margin-bottom: 40px;
-}
+// ---------------------------------------------------------------
+// ---------------------------------------------------------------
+//
+//  Project:		Ubuntu Front-End Download section
+//  Author:			Web Team at Canonical Ltd
+//  Last edited by: Peter Mahnke
```

Comment:
Isn't this and the next line redundant? I don't think there's much chance that we'll remember to keep this up-to-date. Also, we have version control.